source/sync.txt

```diff
@@ -8,20 +8,20 @@ Realm Sync Overview
    :titlesonly:
    :hidden:
    
-   What is Realm Sync? </sync/learn>
+   What is {+sync+}? </sync/learn>
    Configure and Update Your Data Model </sync/configure-your-data-model>
    Define Data Access Patterns </sync/data-access-patterns>
-   Configure Realm Sync </sync/configure-realm-sync>
+   Configure {+sync+} </sync/configure-realm-sync>
```

Comment:
Can we just say "Sync" here instead of using the "Atlas Device Sync" source constant in the ToC? I think "Atlas Device Sync" appearing in the top-level ToC and as the first item in this ToC ("What is Atlas Device Sync") is enough - we use Sync in the other ToC items below and I think that's fine.